### PR TITLE
Get ride of paper-item background

### PR DESCRIPTION
**Pull request by hyyan**
I have added the required css to get ride of the paper-item gray background and added the pointer cursor for the header

### CODE REVIEW

File: paper-collapse-item.html [hunk 1]
```diff
@@ -136,8 +136,14 @@
 		},
 
 		// Private methods
+		/**
+		 * Fired whever the collpase status is changed
```
Comment (ronnyroeller) [inline paper-collapse-item.html:140]: Could you please fix typos? Or better also align naming to "Fired whenever the open status is changed"?

File: paper-collapse-item.html [hunk 2]
```diff
@@ -136,8 +136,14 @@
...
+		 *
+		 * @event toggle
+		 */
 		_toggleOpened: function(e) {
 			this.opened = !this.opened;
+			this.fire('toggle',this);
```
Comment (ronnyroeller) [inline paper-collapse-item.html:146]: For consistency, please add a space before after the comma

**Comment by ronnyroeller** [comment]
Thanks for your contribution, @hyyan 
Exposing the toggle event is a great idea (could you please separate it into another PR?)

I'd prefer to not include the CSS changes for the `:host` as it limits how the element can be used and will also create issues for current users (who might e.g. relay on the element not the `display`). I think it's better if you set these CSS changes on the element that embeds the `<paper-collapse-item>`.
I'd prefer to introduce a mix-in if you need to directly style a sub element.

**Comment by hyyan** [comment]
Sure @ronnyroeller   , I have already sent two PRs one for the `toggle` event  and one for the `style` , if you think that the style PR can be BC break , you are free ignore it .

Thanks in advance

**Pull request closed without merging**